instrumentation/opentelemetry_bandit/lib/opentelemetry_bandit.ex

```diff
@@ -514,12 +514,20 @@ defmodule OpentelemetryBandit do
 
     %{
       HTTPAttributes.http_response_status_code() => status_code,
-      ErrorAttributes.error_type() => meta.exception.__struct__
+      ErrorAttributes.error_type() => error_type(meta.exception)
     }
     |> set_resp_header_attrs(meta.conn, config)
     |> Tracer.set_attributes()
 
     Tracer.end_span()
     OpenTelemetry.Ctx.clear()
   end
+
+  defp error_type(%struct{} = reason) when is_exception(reason) do
+    struct
+  end
+
+  defp error_type(reason) do
+    reason
```

Comment:
Just using `reason` and not `kind` in this case.

Consistent with that `cowboy` does: https://github.com/mimiquate/opentelemetry-erlang-contrib/blob/1ad92865dec8149fb027cdde52b01cc9e66b3121/instrumentation/opentelemetry_cowboy/src/opentelemetry_cowboy.erl#L403-L416.

Reply:
What is `%struct{} = reason`? I don't recognize that syntax and `is_exception` guards for exceptions. We're also looking for the value of `__struct__` which is the exception name.

Reply:
> What is `%struct{} = reason`?

Shorthand for `%{__struct__: struct} = reason`.

Reply:
TIL.

Can you make that more explicit, like `%struct_name{} = reason`?